apps/angular/39-injection-token/src/app/app.config.ts

```diff
@@ -8,5 +9,6 @@ export const appConfig: ApplicationConfig = {
       { path: 'video', loadComponent: () => import('./video.component') },
       { path: 'phone', loadComponent: () => import('./phone.component') },
     ]),
+    { provide: TIMER_VALUE, useValue: DEFAULT_TIMER },
```

Comment:
you can do it even better and define your default value inside the cosntructor of your injectionToken

Reply:
True. Thanks for the observation!

Reply:
Wait, do you mean the Default Token value of the Default Provider Value? Because I didn't try just sending the provider without the useValue field because I thought it was required. 
Guess I need to re read how to set providers in Angular.

Reply:
```ts
export const TIMER = new InjectionToken('call timer', {
  providedIn: 'root',
  factory: () => DEFAULT_TIMER,
});
```